internal/keycloakb/passwordgeneration.go

```diff
@@ -0,0 +1,100 @@
+package keycloakb
+
+import (
```

Comment:
No unit tests here... No necessary mock, they should not be hard to write.

Reply:
DONE